packages/sil-voting-app/src/store/extension/actions.ts

```diff
@@ -1,27 +1,33 @@
+import { TransactionEncoder } from "@iov/encoding";
+
 import { sendGetIdentitiesRequest } from "../../communication/identities";
 import { ExtensionState, SetExtensionStateActionType } from "./reducer";
 
 export async function getExtensionStatus(): Promise<ExtensionState> {
   const identities = await sendGetIdentitiesRequest();
 
   if (!identities) {
-    return { installed: false, connected: false };
+    return { installed: false, connected: false, identity: undefined };
   }
 
   if (Object.keys(identities).length === 0) {
```

Comment:
This is an issue in the old code: `identities` is an array. We should simply use `if (identities.length === 0) {`